e2e-tests/web/wdio.conf.js

```diff
@@ -0,0 +1,33 @@
+exports.config = {
+  runner: 'local',
+  specs: [
+    './web/test/specs/**/*.ts'
+  ],
+  exclude: [],
+  maxInstances: 10,
+
+  capabilities: [{
+    maxInstances: 5,
+    browserName: 'chrome',
+    acceptInsecureCerts: true
```

Comment:
Probably fine as these are just tests, but is this required?

Reply:
removed. was the result of some copy pasting. But might be necessary for native.